Source/non_ip/ai.conf

```diff
@@ -53,3 +53,10 @@ DOMAIN-SUFFIX,clipdrop.co
 DOMAIN-SUFFIX,jasper.ai
 # OpenArt
 DOMAIN-SUFFIX,openart.ai
+# GitHub Copilot
+DOMAIN-SUFFIX,githubcopilot.com
+DOMAIN,copilot-proxy.githubusercontent.com
+# Github Copilot (Claude)
+DOMAIN,copilot-telemetry.githubusercontent.com
+DOMAIN-SUFFIX,exp-tas.com
+DOMAIN,origin-tracker.githubusercontent.com
```

Comment:
This domain does belong to GitHub Copilot, but I myself can access `origin-tracker.githubusercontent.com` in Hong Kong. Are you sure this domain has to be added?